releasenotes/notes/qdrift-as-product-formula-044a37a106a45a47.yaml

```diff
@@ -0,0 +1,6 @@
+---
+prelude: >
+    Reformulate the former QDrift class in qiskit.opflow.evolutions.trotterizations as a ProductFormula instead of a TrotterizationBase.
+other:
+  - |
+    Reformulate the former QDrift class in qiskit.opflow.evolutions.trotterizations as a ProductFormula instead of a TrotterizationBase.
```

Comment:
These would be better as Sphinx cross-references (such as `` :obj:`~qiskit.opflow.evolutions.QDrift` `` and `` :obj:`~qiskit.opflow.evolutions.TrotterizationBase` ``).   Also, `ProductFormula` doesn't appear to be documented at the moment in the API docs, so referring to it might be a bit confusing for users.

Reply:
rewrote the release note and removed `ProductFormula` from it.